packages/client/src/components/Players/Player.tsx

```diff
@@ -10,12 +10,23 @@ import { Box, BoxProps, Grid } from 'theme-ui';
 type Props = {
   player: PlayerState;
   isActive: boolean;
+  isSpectator: boolean;
   togglePlayerActiveCard: React.Dispatch<React.SetStateAction<string | null>>;
   activePlayerCard: string | null;
 };
 
-export const Player = ({ player, isActive, togglePlayerActiveCard, activePlayerCard, ...rest }: Props & BoxProps) => {
+export const Player = ({
+  player,
+  isActive,
+  isSpectator,
+  togglePlayerActiveCard,
+  activePlayerCard,
+  ...rest
+}: Props & BoxProps) => {
   if (!player) {
+    if (isSpectator) {
```

Comment:
I was wrong here, you need this if you are right but maybe you can move it to a ternary, 
```
return !isSpectator ? <InvitePlayer /> : null
```